apps/src/lib/ui/SettingsMenu.jsx

```diff
@@ -0,0 +1,49 @@
+import React, {Component, PropTypes} from 'react';
+import msg from '@cdo/locale';
+import * as assets from '../../code-studio/assets';
+import * as maker from '../kits/maker/toolkit';
+import PopUpMenu from './PopUpMenu';
+
+export default class SettingsMenu extends Component {
+  constructor(props) {
+    super(props);
+
+    // Autobind
+    this.manageAssets = this.manageAssets.bind(this);
+    this.toggleMakerToolkit = this.toggleMakerToolkit.bind(this);
+  }
+
+  static propTypes = {
+    targetPoint: PropTypes.shape({
+      top: PropTypes.number.isRequired,
+      left: PropTypes.number.isRequired,
+    }).isRequired,
+    handleClose: PropTypes.func.isRequired,
+  };
+
+  manageAssets() {
+    this.props.handleClose();
+    assets.showAssetManager();
+  }
+
+  toggleMakerToolkit() {
+    this.props.handleClose();
+    window.dashboard.project.toggleMakerEnabled();
```

Comment:
I think we should be able to import project instead of using `window.dashboard.project`